strabo/static/public_styles/popup.css

```diff
@@ -10,31 +28,31 @@
     z-index: 100000; /* goes above leaflet link */
 }
 .popup-foreground {
+    position:absolute;
     height: 100%;
-    /* size of box on larger screens
-    Dependencies: .text-section:font-size*/
+    /* size of box on larger screens*/
     max-height: 800px;
     max-width: 700px;
-    /*differenciates slightly from normal background*/
+    /*differenciates popup background slightly from normal background*/
```

Comment:
Given that we're doing spelling correction, we should change this to "differentiates."